contracts/Baal.sol

```diff
@@ -495,28 +475,50 @@ contract Baal is Executor, Initializable {
         );
     }
 
-    /// @notice Baal-only function to mint shares.
+    /// @notice Baal-or-shaman-only function to mint shares.
     function mintShares(address[] calldata to, uint96[] calldata amount)
         external
-        baalOnly
+        baalOrShamanOnly
     {
         require(to.length == amount.length, "!array parity"); /*check array lengths match*/
         for (uint256 i = 0; i < to.length; i++) {
             _mintShares(to[i], amount[i]); /*grant `to` `amount` `shares`*/
         }
     }
 
-    /// @notice Baal-only function to mint loot.
+        /// @notice Baal-or-shaman-only function to burn shares.
+    function burnShares(address[] calldata to, uint96[] calldata amount)
+        external
+        baalOrShamanOnly
+    {
+        require(to.length == amount.length, "!array parity"); /*check array lengths match*/
+        for (uint256 i = 0; i < to.length; i++) {
+            _burnShares(to[i], amount[i]); /*grant `to` `amount` `shares`*/
+        }
+    }
+
+    /// @notice Baal-or-shaman-only function to mint loot.
     function mintLoot(address[] calldata to, uint96[] calldata amount)
         external
-        baalOnly
+        baalOrShamanOnly
     {
         require(to.length == amount.length, "!array parity"); /*check array lengths match*/
         for (uint256 i = 0; i < to.length; i++) {
             _mintLoot(to[i], amount[i]); /*grant `to` `amount` `shares`*/
         }
     }
 
+    /// @notice Baal-or-shaman-only function to burn loot.
+    function burnLoot(address[] calldata to, uint96[] calldata amount)
+        external
+        baalOrShamanOnly
+    {
+        require(to.length == amount.length, "!array parity"); /*check array lengths match*/
+        for (uint256 i = 0; i < to.length; i++) {
+            _burnLoot(to[i], amount[i]); /*grant `to` `amount` `shares`*/
+        }
+    }
+
     /// @notice Baal-only function to convert shares to loot.
     function convertSharesToLoot(address to) external baalOnly {
```

Comment:
How important is this function if we could also just chain together _burnShares and _mintLoot in a regular proposal?